### ISSUE TITLE
require of sproutcore-handlebars throwing a js error

### ISSUE THREAD

**Issue by crofty** [bpm/bpm#39]
I'm trying to get up and running with bpm and am getting issues whenever I try and require the sproutcore-handlebars package.

I've an example of what I'm doing at: https://github.com/crofty/example_bpm

The main.js file is very simple, it just requires all of sproutcore, however, when I do this the qunit tests dont run.  If you look in the browser console the error is:

```
 Uncaught TypeError: Object #<Object> has no method 'template'
```

If I just `require('sproutcore-runtime')` then things work as expected and the tests pass.  However, as soon as I also try and `require('sproutcore-handlebars')`, I get the same error.

You can see from the files which versions I'm using:
SC: 2.0.beta3
BPM: 1.0.0.beta13

**Comment by crofty** [comment]
This could have been a sproutcore rather than a bpm error.  I cloned the lastest sproutcore code and copied the `packages` directory into the example_bpm project.  The tests then work as expected.

Looks like there was an issue in the released SC 2.0.beta3 that prevented it working with bpm/spade.

**Comment by wagenet** [comment]
Closing for now then. Let me know if you have further issues.
